CMSgov/design-system
# Bugfix/master ts export updates

## Summary
Applying changes from these two TS export update PRs to master. Have to do this manually since other PRs would introduce development code that is not ready

PRs for reference:
https://github.com/CMSgov/design-system/pull/1118
https://github.com/CMSgov/design-system/pull/1099/files

## How to test
Tests and build should pass. You can compare file changes to those in the PRs to ensure nothing got missed.
